webapp/commonComponents/hooks/utils.js

```diff
@@ -0,0 +1,12 @@
+export const makeMultipart = fn => (url, data = {}, config = {}) =>
```

Comment:
where is this used besides useAsyncMultipartPutRequest.js  ? why does it need to be in utils?